NitroxClient/GameLogic/EquipmentSlots.cs

```diff
@@ -25,17 +25,17 @@ public void BroadcastEquip(Pickupable pickupable, GameObject owner, string slot)
         {
             string ownerGuid = GuidHelper.GetGuid(owner);
             string itemGuid = GuidHelper.GetGuid(pickupable.gameObject);
-            pickupable.gameObject.transform.SetParent(null); // On Deserialized Function Try to find non-existent Parent set null to prevent that bug
+            pickupable.gameObject.transform.SetParent(owner.transform); // On Deserialized Function Try to find non-existent Parent set null to prevent that bug
```

Comment:
Not sure if this is the correct solution.  We intentionally remove the parent so that it deserializes with no parent (other clients that receive an equipment event would not be able to deserialize as the parent doesn't exist).  I think the correct solution would be to keep the parent set to null then at the end of this function reset the parent.  Also add commenting on why we are doing it (the current comment is not clear)

Reply:
1 - I didn't put the comment there. it was there when I started. 
2 - I'll check and see if we can move / change the set parent functions so that it is still null for the packet send